behavior_tests/src/ngt-invalid-json7/compile_commands.json

```diff
@@ -2,6 +2,9 @@
     {
         "command": "nvcc hello1.c",
         "directory": "directory_placeholder",
-        "file": "hello1.c"
+        "file": "hello1.c",
+	"command": "nvcc hello2.c",
+        "directory": "directory_placeholder",
+        "file": "hello2.c"
```

Comment:
Line6~Line8 are duplicated with Line3~line5,  pls end a entry for Line6~8, and make sure the content of compile_comands.json valid.